circle.yml

```diff
@@ -24,10 +24,11 @@ dependencies:
 
 test:
   override:
-    - make clean test test-coverage
+    - echo "ignore unit tests in circleCI"
+    # - make clean test test-coverage
     # workaround - make html returns 0 even if examples fail to build
     # (see https://github.com/sphinx-gallery/sphinx-gallery/issues/45)
-    - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
+    # - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
```

Comment:
> Even I was thinking the same. But I was confused because this is a change in
> circle.yml and appveyor build was failing.

appveyor has a lot of spurious failures :(